app/tutorials/weather-warning/src/test/java/WeatherWarningAppTest.java

```diff
@@ -0,0 +1,167 @@
+/*
+ * Copyright (c) 2020 Fraunhofer FOKUS and others. All rights reserved.
```

Comment:
You could update your template to use 2021 now ;)

Reply:
true :D